src/packagedcode/rubygems.py

```diff
@@ -32,6 +32,8 @@
 from os.path import abspath
 from os.path import expanduser
 
+import io
+import re
 import attr
 import saneyaml
 from packageurl import PackageURL
```

Comment:
Please sort your imports alphabetically in the following way:
```
import built-in-python-libs

import 3rd-party-python-libs

import scancode-project-imports
```

Reply:
Done :)